UnitTests/ClientTest/TrajectoryManagerTest.cpp

```diff
@@ -0,0 +1,182 @@
+#include "TrajectoryManagerTest.h"
+
+#include "testUtilites.h"
+#include "TrajectoryManager.h"
+
+/**
+* \brief Namespace scope to test project.
+*/
+namespace clientTests
+{
+
+void TrajectoryManagerTest::positionalTrajectoryBuilding()
+{
+    const danila::TrajectoryManager trajectoryManager;
+    const std::vector<vasily::RobotData> result
+    {
+        { { 1, 1, 1, 1, 1, 1 }, { 10, 2, 0 } },
+        { { 2, 2, 2, 2, 2, 2 }, { 10, 2, 0 } },
+        { { 3, 3, 3, 3, 3, 3 }, { 10, 2, 0 } },
+        { { 4, 4, 4, 4, 4, 4 }, { 10, 2, 0 } }
+    };
+
+    const vasily::ParsedResult parsedResult{ 0, 3, true, result };
+
+    const std::vector<vasily::RobotData> trajectory = 
+        trajectoryManager.PositionalMovement(parsedResult);
+
+    Assert::AreEqual(result.size(), trajectory.size(), L"Not equal numer of positions");
```

Comment:
**(note, repeadetly, grammar nazi):** num**b**er

Reply:
I fixed it.